store-pg/src/test/scala/com/socrata/pg/store/ResyncTest.scala

```diff
@@ -69,5 +69,101 @@ class ResyncTest extends PGSecondaryTestBase with PGStoreTestBase with PGSeconda
     }
   }
 
+  test("handle drop copy") {
+    withPgu() { pgu =>
+      val pgs = new PGSecondary(config)
+      val secondaryDatasetInfo = DatasetInfo("monkey", "locale", "obfuscate".getBytes)
+
+      val snapshottedCopy = CopyInfo(new CopyId(123), 1, LifecycleStage.Snapshotted, 24, new DateTime())
+      val publishedCopy = CopyInfo(new CopyId(123), 2, LifecycleStage.Published, 55, new DateTime())
+      val unpublishedCopy = CopyInfo(new CopyId(123), 3, LifecycleStage.Unpublished, 57, new DateTime())
+      val discardedCopy = CopyInfo(new CopyId(123), 3, LifecycleStage.Discarded, 58, new DateTime())
```

Comment:
Yeah, when you discard a working copy it maintains its copy number. Or that was what I was trying to represent with my test.